config/sync/user.role.authenticated.yml

```diff
@@ -40,6 +40,7 @@ permissions:
   - 'use text format os2loop_question_answer_rich_text'
   - 'use text format os2loop_question_plain_text'
   - 'use text format os2loop_question_rich_text'
+  - 'use text format os2loop_section'
```

Comment:
This should contain `os2loop_section_page`.

Reply:
Fixed